### PR TITLE
Fix Merriam Wester dictionary

### PR DESCRIPTION
**Pull request by rickliujh**
Merriam Wester dictionary has not working for a long time, sense I have highly demand to use it I decide to fix it.

Engine covered by unit test, Firefox browser is tested.

### CODE REVIEW

File: src/components/dictionaries/merriamwebster/View.tsx [hunk 1]
```diff
@@ -1,61 +1,140 @@
...
+                    <div className="mw-mg-sign"> {o + 1}</div>
+                    <div className="mw-mg-line"></div>
+                  </div>
+
+                  <div className="mw-mg-right">
+                    {means.map((mean, k) => (
+                      <div key={'mw-meaning-' + k} className="mw-mean-area">
+                        {(mean.examples || mean.explaining) &&
+                          means.length > 1 && (
+                            <span className="mw-mean-sign">{alphabet[k]}</span>
```
Comment (crimx) [inline src/components/dictionaries/merriamwebster/View.tsx]: Can `alphabet[k]` be replaced with `String.fromCharCode(k - 97)`?

**Review by crimx — CHANGES_REQUESTED** [review]
Thanks for the PR!

File: src/components/dictionaries/merriamwebster/View.tsx (hunk 1 above)
Comment (rickliujh) [reply src/components/dictionaries/merriamwebster/View.tsx]: thanks for point it out, done.

**Review by crimx — APPROVED** [review]

**Comment by rickliujh** [comment]
Hi @crimx, I'd like to ask when to release the fix since my local build version wasn't works well on its setting language in Edge browser. Anything that I can help please let me know.

**Pull request merged**